### DISCUSSION TITLE
Add CodeFormer to improve face generation

### DISCUSSION THREAD

**Discussion by kamerosoul** [comfyanonymous/ComfyUI#228] [Ideas]
I like ComfyUI but I miss CodeFormer. Please add it to improve face generation or if it is already implemented let me know install or copy from webui or if there is another option also let me know.

Thanks

**Comment by WASasquatch** [comment]
I honestly not sure they can package codeformer with ComfyUI, as this allows commercial usage of it's code and outputs, and CodeFormer does not, it's research only, and non-commercial. Licenses can't contradict each other.
